src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -241,10 +243,10 @@ public override IProjectTree FindByPath(IProjectTree root, string path)
             // from ParseCanonicalName, which in turn called from progression's SourceFileToHierarchyItem where
             // LocalPath of the Uri is passed, not full Uri during search. Thus here we need to append file:/// 
             // if it is not there.
-            if (!path.StartsWith("file:///"))
-            {
-                path = "file:///" + path.Trim('/');
-            }
+            //if (!path.StartsWith("file:///"))
```

Comment:
Let's not leave commented code in the code base. If this needed to be commented out because of a bug, please file one.